subschemas/substrate-api-proxy/src/resolvers/Query/crowdloan.ts

```diff
@@ -48,3 +50,95 @@ export async function crowdloanSummary(
     totalFunds: data.funds.length,
   };
 }
+
+interface CrowdloanInfo extends Omit<Crowdloan, 'depositor'> {
+  depositor: PartialDepositor;
+}
+
+export type PartialDepositor = {
+  address: string;
+};
+
+export async function activeCrowdloans(
+  _: Record<string, never>,
+  __: Record<string, never>,
+  {api}: Context,
+): Promise<CrowdloanInfo[]> {
+  const [paraIdKeys, bestNumber] = await Promise.all([
+    api.query.crowdloan?.funds?.keys<[ParaId]>(),
+    api.derive.chain.bestNumber(),
+  ]);
+
+  const paraIds = paraIdKeys.map(({args: [paraId]}) => paraId);
+  const data = await getFunds(paraIds, bestNumber, api);
+  const leasePeriod = await getLeasePeriod(api);
+  const activeFunds = extractActiveFunds(data.funds, leasePeriod);
+
+  const funds = activeFunds.map(async (fund) => {
+    const {info, isWinner, paraId} = fund;
+    const {end, firstPeriod, lastPeriod, cap, raised, depositor} = info;
+    const blocksLeft = end.gt(bestNumber) ? end.sub(bestNumber) : BN_ZERO;
+    const status = isWinner ? 'Winner' : 'Active';
+    const ending = getBlockTime(api, blocksLeft);
+
+    const contribution = await api.derive.crowdloan.contributions(paraId);
```

Comment:
Should we make it a separate resolver of it's own named `crowdloanContribution` and use that here instead of this?
@franjs

Reply:
We can have a resolver for this, so we can used it here and in the crowdloan detail